data_model.md

```diff
@@ -371,11 +378,16 @@ from them. CSV columns must be delimited by “,” and rows must be
 delimited by “\n”. The CSV must contain a header line of
 “timestamp,value,quality_flag” followed by the data rows.
 
+The following block is an example of a valid CSV upload.
+
 ```
 # optional header, ignored by Solar Forecast Arbiter
 timestamp,value,quality_flag
-2018-11-22T12:01:48Z,10.23,0
-2018-11-22T12:07:38Z,10.67,0
+2019-03-01T12:00:00Z,5,0
+2019-03-01T12:00:01Z,,1
+2019-03-01T12:00:02Z,5,0
+2019-03-01T12:00:03Z,NaN,1
+2019-03-01T12:00:04Z,NULL,1
```

Comment:
I don't think we should have an example with 1 second interval, maybe 1min or 5min is ok